nx-cloud/values.yaml

```diff
@@ -1,11 +1,94 @@
-image:
```

Comment:
Technically removal of `image.tag` will be a braking change for this update. If you are ok with this change, I will go and update remaining READMEs and examples.